src/js/scripts.js

```diff
@@ -387,6 +388,9 @@ $(document).ready(() => {
 
   svg4everybody();
 
+
+  easyAutocomplete();
```

Comment:
Is there a need for this? easyAutocomplete() is already called in the homepage template. 

I was getting an error, but when I removed this it seemed to work.

Reply:
I was having a lot of issues initially trying to get this to work so I think it might have been left over from one of the things I tried.

Reply:
ok